colabs/mosaicml/MosaicML_Composer_and_wandb.ipynb

```diff
@@ -0,0 +1,765 @@
+{
```

Comment:
<p>I would push this to the bottom in an "advanced" section. To start I think we should just showcase a super simple and quick to understand example of how the logger works. Then we can show off more advanced features like this custom callback for Tables</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
done